lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -539,9 +588,12 @@ class MongoClientInterface {
         } else if (params && params.versionId === '') {
             return this.putObjectVerCase2(c, bucketName, objName, objVal,
                                           params, log, cb);
-        } else if (params && params.versionId) {
+        } else if (params && params.versionId && !params.usePHD) {
```

Comment:
I think it's worth spelling out the param as `usePlaceholder`.

Reply:
It's actually used as PHD throughout our versioning logic. I think its ok.